### PR TITLE
Update tasks.py

### PR DESCRIPTION
**Pull request by smk4664**
Remove --config option from make-migrations check.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -183,7 +243,7 @@ jobs:
       - name: "Upload binaries to release"
         uses: "svenstaro/upload-release-action@v2"
         with:
-          repo_token: "${{ secrets.GH_NAUTOBOT_BOT_TOKEN }}"
+          repo_token: "${{ secrets.NTC_GITHUB_TOKEN }}"  # use GH_NAUTOBOT_BOT_TOKEN for Nautobot Org repos.
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:246]:
```suggestion
          repo_token: "${{ secrets.GH_NAUTOBOT_BOT_TOKEN }}"  # use GH_NAUTOBOT_BOT_TOKEN for Nautobot Org repos.
```

File: changes/130.housekeeping [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Remove --config option from make-migrations check.
```
Comment (gsnider2195) [inline changes/130.housekeeping:1]:
```suggestion
Fixed `invoke check-migrations`.
```

File: changes/130.housekeeping [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+Update to cookiecutter-nautobot-app ltm-1.6
```
Comment (gsnider2195) [inline changes/130.housekeeping:2]:
```suggestion
Updated to cookiecutter-nautobot-app ltm-1.6.
```

File: development/docker-compose.base.yml [hunk 4]
```diff
@@ -7,13 +7,12 @@ x-nautobot-build: &nautobot-build
     context: "../"
     dockerfile: "development/Dockerfile"
 x-nautobot-base: &nautobot-base
-  image: "nautobot-floor-plan/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
+  image: "nautobot-device-lifecycle-mgmt/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
```
Comment (gsnider2195) [inline development/docker-compose.base.yml:10]:
```suggestion
  image: "nautobot-floor-plan/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
```

File: development/docker-compose.dev.yml [hunk 5]
```diff
@@ -12,13 +11,19 @@ services:
...
+      retries: 3
+      test: ["CMD", "true"]  # Due to layering, disable: true won't work. Instead, change the test
   docs:
     entrypoint: "mkdocs serve -v -a 0.0.0.0:8080"
     ports:
       - "8001:8080"
     volumes:
       - "../:/source"
-    image: "nautobot-floor-plan/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
+    image: "nautobot-device-lifecycle-mgmt/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
```
Comment (gsnider2195) [inline development/docker-compose.dev.yml:26]:
```suggestion
    image: "nautobot-floor-plan/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
```

File: tasks.py [hunk 6]
```diff
@@ -396,9 +677,19 @@ def check_migrations(context):
...
         "buffer": "Discard output from passing tests",
+        "pattern": "Run specific test methods, classes, or modules instead of all tests",
+        "verbose": "Enable verbose test output.",
     }
 )
-def unittest(context, keepdb=False, label="nautobot_floor_plan", failfast=False, buffer=True):
+def unittest(
+    context,
+    keepdb=False,
+    label="nautobot_device_lifecycle_mgmt",
```
Comment (gsnider2195) [inline tasks.py:687]:
```suggestion
    label="nautobot_floor_plan",
```

File: development/docker-compose.base.yml (hunk 4 above)
Comment (smk4664) [reply development/docker-compose.base.yml:10]: Good catch, I didn't notice that one.

**Pull request merged**